go.mod

```diff
@@ -5,6 +5,7 @@ go 1.15
 replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309 // Required by Helm
 
 replace (
+	github.com/apache/thrift => github.com/apache/thrift v0.14.1 // For fixing https://bugzilla.redhat.com/show_bug.cgi?id=1928809
```

Comment:
Whats the reasoning behind putting this in the replace directive vs the require directive?

Reply:
`go get -u github.com/apache/thrift@v0.14.1` will add it to `require`. However the next `go mod tidy` will remove it as we don't directly use it. LMK if I am missing something.